test/unit/models/forms/LiabilityDetailsFormTest.scala

```diff
@@ -121,14 +128,14 @@ class LiabilityDetailsFormTest extends ModelsBaseSpec {
         val form = LiabilityDetailsForm.liabilityDetailsForm(sampleCase)
 
         form.hasErrors shouldBe false
-        form.data      shouldBe params.mapValues(v => v.head)
+        form.data      shouldBe params.map { case (fst, snd) => fst -> snd.head }
```

Comment:
Same with the others as it can just be `params.view.mapValues(v => v.head).toMap`. Thanks